container-build-task/task.json

```diff
@@ -7,9 +7,9 @@
   "description": "Build a Docker image with Google Cloud Container Builder.",
   "helpMarkDown": "Use [Google Cloud Container Builder](https://cloud.google.com/container-builder/docs) to build a Docker image and place it in [Google Container Registry](https://cloud.google.com/container-registry/docs).",
   "version": {
-    "Major": 0,
+    "Major": 1,
```

Comment:
Can we not update to a major version of 1? This sort of implies that this is a GA version.

Reply:
Unfortunately, VSTS will auto-upgrade tasks that are incremented by less than a Major version. To enable selecting different versions, we have to upgrade the Major version. Fortunately, the task versions are only visible when using the task, and the extension as a whole (the version number visible on the extension page) is still just 0.0.4, indicating a prerelease version.